uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_manage-leave-requests.scss

```diff
@@ -57,20 +84,52 @@
  * @TODO remove this patch when Shoreditch theme is migrated to SSP
  */
 .chr_manage_leave_requests {
-  .btn.ui-select-toggle {
-    padding: 4px 10px !important;
-    text-transform: none !important;
+  &[data-role="admin"] {
+    .ui-select-toggle {
```

Comment:
Please remove the selector altogether. And the `box-shadow`is a theme problem, so if we need to fix it we should do it only in shoreditch. But I would say that its not a big issue(fixing in Shoreditch) and can be skipped for now.